### PR TITLE
ROC-31 profile updates deletes

### PR DESCRIPTION
**Pull request by squaill**
POST/PATCHing when events are triggered because these are technically two different forms and I didn't want to deal with submitting them with one button on a parent component because I'm dumb and bad at React. Thank you for coming to my TED talk.

### CODE REVIEW

**Comment by LarrySchultheis** [comment]
That's a lot of words

File: src/components/UserProfile/PersonalDetailsForm.js [hunk 1]
```diff
@@ -11,21 +11,20 @@ import MenuItem from '@mui/material/MenuItem';
...
-    console.log(gender);
     return (
-        <>
+        <form id="userForm">
             <Typography sx={{mt: "2rem", mb:"2rem"}} color='#8A8A8A' variant="h4">Personal Details</Typography>
         <Stack
             component="form"
             alignItems="center"
         >
             <TextField
```
Comment (LarrySchultheis) [inline src/components/UserProfile/PersonalDetailsForm.js:37]: Probably just need an onChange hook in these fields to handle user input. Also need some sort of submit button if we're allowing user's to change their name/gender.

File: src/components/MatchProfile/AllChipsArray.js [hunk 2]
```diff
@@ -5,16 +5,64 @@ import {useEffect, useState} from 'react';
...
-const ListItem = styled('li')(({ theme }) => ({
-  margin: theme.spacing(0.5),
-}));
+const axiosInstance = axios.create({
+  baseURL: "http://localhost:5000",
+  headers: {
+    "Content-Type": "application/json"
+  }
+});
+// export default function MultiSelect(props) {
```
Comment (LarrySchultheis) [inline src/components/MatchProfile/AllChipsArray.js:16]: Don't forget to remove comments if this code isn't gonna be used

**Comment by LarrySchultheis** [comment]
Just a general comment to save you some time: build failure is caused by some eslint warnings

File: src/components/UserProfile/PersonalDetailsForm.js (hunk 1 above)
Comment (LarrySchultheis) [reply src/components/UserProfile/PersonalDetailsForm.js:37]: Also wrote a semi okay input validation pattern per the admin management stuff. MUI form components can be weird. See https://github.com/LarrySchultheis/rockr-ui/pull/14/files (AdminManagement.js) if you wanna copy

**Review by LarrySchultheis — COMMENTED** [review]
ugh never submitted review love github

**Review by LarrySchultheis — APPROVED** [review]

File: src/components/MatchProfile/AllChipsArray.js (hunk 2 above)
Comment (squaill) [reply src/components/MatchProfile/AllChipsArray.js:16]: yeah, I plan to integrate that so I just left it. I want users to be able to add new chips from an autocomplete multi-select dropdown. Probably didn't need to leave it in there but I didn't want to go searching for it in the mui docs again

File: src/components/UserProfile/PersonalDetailsForm.js (hunk 1 above)
Comment (squaill) [reply src/components/UserProfile/PersonalDetailsForm.js:37]: I am confusion. Are you saying I should use onChange for the fields in this form?

**Pull request merged**